packages/yoroi-ergo-connector/src/inject.js

```diff
@@ -32,7 +32,7 @@ function ergo_check_read_access() {
 function cardano_request_read_access() {
     return new Promise(function(resolve, reject) {
         window.postMessage({
-            type: "yoroi_connect_request/cardano",
+            type: "connector_connect_request/cardano",
```

Comment:
It's probably not too important but `window.postMessage` is shared amongst all wallets trying to implement this so we might not want too generic of a message to send, maybe we should have both have "yoroi" somewhere in the name to be safer?